### PR TITLE
Add base64-to-file extension

### PR DESCRIPTION
**Pull request by iamforrest**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by arian81** [comment]
Hey you should change the icon of your extension, it's going to get rejected per docs.

File: extensions/base64-to-file/assets/command-icon.png
Comment (arian81) [inline extensions/base64-to-file/assets/command-icon.png:1]: You can't use default icon change it to something else. https://icon.ray.so/ This website is helpful for generating icons.

Comment (iamforrest) [reply extensions/base64-to-file/assets/command-icon.png:1]: added my own icon

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Maybe add a preference which uses [directory](https://developers.raycast.com/api-reference/preferences#return) so the user is able to set a standard path for the output?

- Is this only for images and what format? Tried entering `SG93IGFyZSB5b3UgdG9kYXk=` which added a file, but the file is corrupted

- Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by erics118** [comment]
There are two extensions that I think do similar things:
https://www.raycast.com/DanielSinclair/base64
https://www.raycast.com/eth-p/encoding-tools

I'm not too sure, but it might be better to contribute to one of those extensions instead of making a new one.

**Comment by pernielsentikaer** [comment]
It could also be a way to go yes, are you still around @iamforrest

**Comment by iamforrest** [comment]
I've attached three screenshots directly here. However, sorry that I'm unsure of the appropriate directory within the codebase to place these images.

Xnip2024-03-01_18-52-18
Xnip2024-03-01_19-01-20
Xnip2024-03-01_19-01-50

**Comment by pernielsentikaer** [comment]
So, what are you using as base64? Can you share an example so I can test?

**Comment by iamforrest** [comment]
> So, what are you using as base64? Can you share an example so I can test?

here is an example:


[base64.txt](https://github.com/raycast/extensions/files/14471469/base64.txt)

[Raycast Developers.pdf](https://github.com/raycast/extensions/files/14471466/Raycast.Developers.pdf)

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**